src/translation/en.json

```diff
@@ -57,6 +60,7 @@
     "herounit.pleaseregister": "Hello, I am the Loremaster Cho. Could you introduce yourself ?",
     "issue": "Report a bug",
     "mount": "Mount",
+    "paypal": "Paypal",
```

Comment:
Why trad paypal if it's the same in english and fr ?

Reply:
No reason for english and french; But it could make a sense if we have another language like korean etc...

Reply:
Will change tonight the id "paypal" to something like "help.a.support.payment"